### PR TITLE
Feature/issue 14 create free basics controller

### PR DESCRIPTION
**Pull request by olpie101**
based on #14 

currently only works by going to `<base-url>/freebasics/add` and `<base-url>/freebasics/<id>`

### CODE REVIEW

**Comment by olpie101** [comment]
@miltontony ready for review :smile:

File: controllers/freebasics/forms.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from controllers.docker.forms import DockerControllerForm
+from controllers.freebasics.models import FreeBasicsController
+from django import forms
+from django.forms.utils import ErrorList
+
+TEMPLATE_CHOICES = (
```
Comment (miltontony) [inline controllers/freebasics/forms.py]: You can move these to `line 20` so that they can be referenced using `FeeBasicsControllerForm.TEMPLATE_CHOICES`

Comment (miltontony) [inline controllers/freebasics/forms.py]: Actually, can you move this to `models.py` as part of the `FreeBasicsController` and use that as the choices there too. You can reference it from here using `FreeBasicsController.TEMPLATE_CHOICES`

**Comment by miltontony** [comment]
:+1: :cake: looks great!

**Pull request merged**